docs/usage/shoot_pod_autoscaling_best_practices.md

```diff
@@ -0,0 +1,337 @@
+
+### VPA Updater Options
+
+You can read up the [full VPA updater options](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/FAQ.md#what-are-the-parameters-to-vpa-updater) online and set some of them conveniently in your [Gardener shoot cluster spec](https://github.com/gardener/gardener/blob/957e4c7/example/90-shoot.yaml#L298-L307):
+- `evictAfterOOMThreshold` (default 10m): Pods where at least one container OOMs within this time period since its start will be actively evicted, which will implicitly apply the new target recommendation that will have been [bumped up after `OOMKill`](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/README.md#custom-memory-bump-up-after-oomkill). Please note, the kubelet may evict pods even before an OOM, but only if `kube-reserved` is underrun, i.e. [node-level resources are running low](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#pod-selection-for-kubelet-eviction). In these cases, eviction will happen first by pod priority and second by how much the usage overruns the requests.
+- `evictionTolerance` (default 50%): Defines a threshold below which no further eligible pod will be evited anymore, i.e. limits how many eligible pod may be in eviction in parallel (but at least 1). The [threshold is computed as follows](https://github.com/kubernetes/autoscaler/blob/4d0511363eeeff657119797dd8d26e851dcc3459/vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction.go#L108-L117): `running - evicted > replicas - tolerance`. Example: 10 replicas, 9 running, 8 eligible for eviction, 20% tolerance with 10 replicas which amounts to 2 pods, and no pod evicted in this round yet, then `9 - 0 > 10 - 2` is true and a pod would be evicted, but the next one would be in violation as `9 - 1 = 10 - 2` and no further pod would be evicted anymore in this round.
+- `evictionRateBurst` (default 1): Defines how many eligible pods may be evicted in one go.
+- `evictionRateLimit` (default disabled): Defines how many eligible pods may be evicted per second (a value of 0 or -1 disables the rate limiting).
+
+There are a few more configurable options of lesser interest:
+- `updaterInterval` (default 1m): How often VPA evicts the pods.
+
+There are many more options that you can only configure if you deploy your own VPA and which we will not discuss here, but you can check them out [here](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/FAQ.md#what-are-the-parameters-to-vpa-updater).
+
+## Considerations When Using VPA
+
+- **Initial Resource Estimates:** VPA requires historical resource usage data to base its recommendations on. Until they kick in, your initial resource requests apply and should be sensible.
+- **Pod Disruption:** When VPA adjusts the resources for a pod, it may need to "recreate" the pod, which can cause temporary disruptions. This should be taken into account.
```

Comment:
Here, I'd put much more focus on the importance of having a well-defined PDB. This seems just like hinting at it.
Also, we can mention `unhealthyPodEvictionPolicy: AlwaysAllow` specifically here https://kubernetes.io/docs/tasks/run-application/configure-pdb/#unhealthy-pod-eviction-policy to avoid problems where a tightly defined PDB prevents VPA from evicting and applying new recommendations when e.g. one Pod is already in OOMkill CrashLoopBackoff

Reply:
Done (reusing the paragraph from the HA guide in which I also mentioned `AlwaysAllow`).